tests/src/Kernel/UnitEntityTest.php

```diff
@@ -49,4 +50,14 @@ public function testEntityDeletion() : void {
     $this->assertNotEquals(NULL, Unit::load(1));
   }
 
+  /**
+   * Test provided languages.
+   *
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   */
+  public function testProvidedLanguages() : void {
+    $entity = $this->getEntity(1);
+    $this->assertEquals('Finnish, English, Arabic', $entity->getProvidedLanguageNames());
```

Comment:
We should test the translated value too. See https://github.com/City-of-Helsinki/drupal-module-helfi-api-base/blob/main/tests/src/Traits/LanguageManagerTrait.php